packages/camera/camera_android_camerax/lib/src/rotated_preview.dart

```diff
@@ -0,0 +1,117 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+/// Widget that rotates the camera preview to be upright according to the
+/// current device orientation.
+final class RotatedPreview extends StatefulWidget {
```

Comment:
nit: We could use `@internal` from `package:meta` to help others understand "this is an implementation detail".

As a bonus we'll get a lint warning if we ever accidentally expose this in the public API.

https://pub.dev/documentation/meta/latest/meta/internal-constant.html